app/migration/fedora_migrate/class_ordered_repository_migrator.rb

```diff
@@ -127,5 +127,12 @@ def qualifying_object(object, klass)
         name = object.pid.split(/:/).first
         return object if (name.match(namespace) && object.models.include?("info:fedora/afmodel:#{klass.name.gsub(/(::)/, '_')}"))
       end
+
+      def construct_migrate_from_uri(soure)
+        result = FedoraMigrate.fedora_config.credentials[:url]
+        result += '/' unless result.ends_with? '/'
+        result += source.pid
+        result
+      end
```

Comment:
Could do this instead:
'''def construct_migrate_from_uri(soure)
  RDF::URI.new(FedoraMigrate.fedora_config.credentials[:url]) / source.pid
end'''